content/Hardware Support/Generic/Update-the-firmware-for-WiFiNINA-and-WiFi101.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Update the firmware for WiFiNINA and WiFi101"
+id: 360013896579
+---
+
+Learn how to update the firmware for WiFiNINA and WiFi101 using the Firmware Updater tool.
```

Comment:
Since we also cover IoT Cloud as an option, in which case the term "Firmware Updater" doesn't really enter into it for the user, we can avoid the use of it here.

```suggestion
Learn how to update the firmware for the WiFi101 and WiFiNINA communication libraries.
```

I would consider including the supported boards here and dropping the "compatible boards" section:

```suggestion
Learn how to update the firmware for the WiFi101 and WiFiNINA communication libraries used on these boards:

* [Arduino MKR WiFi 1010](https://store.arduino.cc/arduino-mkr-wifi-1010)
* [Arduino MKR Vidor 4000](https://store.arduino.cc/arduino-mkr-vidor-4000)
* [Arduino Nano 33 IoT](https://store.arduino.cc/arduino-nano-33-iot)
* [Arduino UNO WiFi Rev2](https://store.arduino.cc/arduino-uno-wifi-rev2)
* [Arduino Nano RP2040 Connect](https://store.arduino.cc/collections/boards/products/arduino-nano-rp2040-connect)
* [Arduino MKR 1000 WiFi](https://docs.arduino.cc/hardware/mkr-1000-wifi)
```